layouts/shortcodes/translation-status-zh-cn.html

```diff
@@ -1,6 +1,6 @@
 <figure>
     <p>
         <strong>翻译状态：</strong>
-        本文是英文页面 <a href="{{ .Get "raw_link" }}">{{ .Get "raw_title" }}</a> 的翻译，最后翻译时间：{{ .Get "last_tranlation_time" }}，点击<a href="https://github.com/tootsuite/documentation/compare/{{ .Get "raw_commit" }}...master">这里</a>可以查看翻译后页面的改动。
+        本文是英文页面 <a href="{{ .Get "raw_link" }}">{{ .Get "raw_title" }}</a> 的翻译，最后翻译时间：{{ .Get "last_translation_time" }}，点击<a href="https://github.com/tootsuite/documentation/compare/{{ .Get "raw_commit" }}...master">这里</a>可以查看翻译后页面的改动。
```

Comment:
Normally my tooling more or less automatically skips localizations. It didn't here...